frontend/src/store/index.js

```diff
@@ -252,25 +263,34 @@ export const store = new Vuex.Store({
   getters: {
     room_resources(state) {
       if (state.rooms.length > 0) {
-        return state.rooms.map(room =>
+        let rooms = state.rooms.map(room =>
           ({
             id: room.room_id,
             title: room.room_name,
             eventColor: room.color
           })
         )
+        rooms.push({
```

Comment:
This looks like it's only there for testing?

Reply:
The above lines are in the getters, its just mapping the keys of the rooms object as provided  to the keys needed by full-calendar since I need that information over multiple components and it would also be useful not to mutate the original data.

Reply:
I get that for the `let rooms = state.rooms.map(room =>` line, it's this `rooms.push({` that looks like a test something to me. Otherwise we're going to have the 

```
{
  id: 2,
  title: 'blah',
  eventColor: 'green'
}
```
returned by every call to the `room_resources` getter, yea?

Reply:
didn't notice this when you pointed it out but I since caught it independently.  yes, the part you were actually referring to was plugged data for testing.  it is there no more.